apollos-church-api/config.yml

```diff
@@ -117,23 +95,23 @@ TABS:
         title: 'Read More'
         relatedNode:
           __typename: UniversalContentItem
-          id: 21
+          id: a32b6019-e817-4b6e-904e-e002e2014485
     - algorithms: [DAILY_PRAYER]
       isCard: false
       type: PrayerList
       title: Daily Prayer
     - algorithms:
         - type: CAMPAIGN_ITEMS
           channelIds:
-            - 9
+            - 0a269d99-2e1a-4fd3-aedf-7e0abc10e948
       type: HeroList
       isFeatured: true
       subtitle: Featured
     - title: FOR YOU
       algorithms: [SERMON_CHILDREN, PERSONA_FEED]
       subtitle: Explore what God calls you to today
       type: ActionList
-    - algorithms: [PERSONA_FEED]
```

Comment:
I checked the `personaFeed` query and it seems ok, is it broken?

Reply:
I think I was frustrated when testing that there were two persona feeds, so I removed one of them :) we can put it back

Reply:
nah, I'm good leaving like this, no redundancy. just wanted to make sure it wasn't broken